packages/ember-views/lib/system/jquery.js

```diff
@@ -1,13 +1,14 @@
 import { context } from 'ember-environment';
 import { hasDOM } from 'ember-browser-environment';
+import { ENV } from 'ember-environment';
 
 let jQuery;
-export let jQueryDisabled = false;
+export let jQueryDisabled = ENV._JQUERY_INTEGRATION === false;
 
 if (hasDOM) {
   jQuery = context.imports.jQuery;
 
-  if (jQuery) {
+  if (!jQueryDisabled && jQuery) {
```

Comment:
Not sure how this is all connected, but I see a few modules importing `jQuery` from this module, without checking for `jQueryDisabled`. Before this change, there was no case possible that `jQuery` was available while `jQueryDisabled` is true. This is now possible, with the flag being false but the jQuery global still available. So should we override `jQuery` to `undefined`, if `jQueryDisabled` is true?

Reply:
@simonihmig I think you might be right. Not if the user has explicitly disabled jQuery, `Ember.$` will be undefined, even if jQuery happens to be included in the page in some other way.

Reply:
Ya, I agree. I think that we need to change all of our guards to be checking `jQueryDisabled` instead of the default export of this module.

tldr; it should absolutely be possible to tell Ember to treat jQuery as disabled _even if_ `window.$` is present...

Reply:
@rwjblue I already did in this PR

Reply:
Doh! sorry :( I guess I thought we had more places that did the conditional...